letsencrypt/config.yaml

```diff
@@ -85,4 +85,8 @@ schema:
     sakuracloud_api_token: str?
     transip_api_key: str?
     transip_username: str?
+    inwx_url: str?
```

Comment:
Does this truly need to be a config option? If every user is just going to put `https://api.domrobot.com/xmlrpc/` then don't make it an option in the addon, set it for users instead. If many users are expected to need to change this then we can leave it.

Not every library option needs to be an addon option. If we can default something for people then we should do that.

Reply:
good idea